src/mapproxyUtils/mapproxySeed.ts

```diff
@@ -78,6 +82,28 @@ export class MapproxySeed {
     }
   }
 
+  public addTimeMinuteBuffer(dataTimeStr: string): string {
+    const secondsInMin = 60;
```

Comment:
can this consts be defined in a separate "consts" file or at least in as class static members?

Reply:
done